trello_config.py

```diff
@@ -0,0 +1,13 @@
+trello_root_url = 'https://api.trello.com/1'
+trello_board_id = '5ef8f1926a457142f41ac6ed'
+trello_todo_list_id = '5ef8f19a64a8ef61948d43fd'
```

Comment:
These could be inferred from the list names.

Reply:
Now done. In a production app I'd cache these but for now to keep it simple/stateless I'm looking them up each time.